source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTapGestureRecognizerFactory.h

```diff
@@ -9,9 +9,10 @@
 @interface ACRTapGestureRecognizerFactory:NSObject
 /// instantiates a target for UITapGestureRecognizer object
 /// and instantiate a tap gesture reconginizer with target, and return it
-+ (UITapGestureRecognizer *)getTapGestureRecognizer:(UIView<ACRIContentHoldingView> *)viewGroup
-                                 rootViewController:(UIViewController *)vc
-                                      actionElement:(std::shared_ptr<BaseActionElement> const &)action
-                                             inputs:(NSMutableArray *)inputs
-                                         hostConfig:(std::shared_ptr<HostConfig> const &)config;
++ (UILongPressGestureRecognizer *)getTapGestureRecognizer:(UIView<ACRIContentHoldingView> *)viewGroup
```

Comment:
>UILongPressGestureRecognizer [](start = 3, length = 28)

It seems strange that the `getTapGestureRecognizer` method on the `ACRTapGestureRecognizerFactory` interface returns an object that is not a `UITapGestureRecognizer`. It looks like the interface and method should have their names updated.

Reply:
You are absolutely correct! thanks

---
In reply to: [164589949](https://github.com/Microsoft/AdaptiveCards/pull/1075#discussion_r164589949) [](ancestors = 164589949)